README.md

```diff
@@ -25,6 +25,7 @@ Note: We recommend running these commands in a unix terminal. Windows users can
     cp .env.template .env
     ```
 1. Update the `.env` file with your [Plaid API keys][plaid-keys].
+1. Configure your dashboard with the Oauth redirect URI: `http://localhost:3000/oauth-link` on your [dashboard][plaid-dashboard].
```

Comment:
might sound a little cleaner to just use `dashboard` once like:

Configure the Oauth redirect URI: `http://localhost:3000/oauth-link` on your [dashboard][plaid-dashboard].

Reply:
yup. gotcha